GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -124,7 +125,23 @@ private void UnhandledGVFSExceptionHandler(ITracer tracer, object sender, Unhand
 
         private JsonTracer CreateTracer(GVFSEnlistment enlistment, EventLevel verbosity, Keywords keywords)
         {
-            JsonTracer tracer = new JsonTracer(GVFSConstants.GVFSEtwProviderName, "GVFSMount");
+            string enlistmentId = null;
+            string mountId = null;
+
+            GitProcess git = new GitProcess(enlistment);
+            GitProcess.Result configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
+            if (!configResult.HasErrors)
+            {
+                enlistmentId = configResult.Output.Trim();
+            }
+
+            configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.MountId);

```

Comment:
Why is the mount id coming from a config file? I thought this was supposed to be a new value per mount?

Reply:
I have confirmed it is a new value for each mount process (set in CloneVerb and MountVerb). It's in the git config file so the new built-in Git tracing can also sweep up and include this value for better correlation between Git events and GVFS events.